rules/klayout/drc/testing/run_regression.py

```diff
@@ -32,28 +32,30 @@
     --run_name=<run_name>               Select your run name.
 """
 
+from subprocess import check_call
+
 from docopt import docopt
 import os
 import datetime
 import xml.etree.ElementTree as ET
 import csv
 import time
 import re
-
-from sympy import arg
+import pandas as pd
+import logging
 
 def call_regression(rule_deck_path, path):
+
     t0 = time.time()
     marker_gen = []
-    rules =[]
-    ly = 0
+    rules      = []
```

Comment:
can you make sure this follow https://peps.python.org/pep-0008/ ?

Reply:
@atorkmabrains Will check how we can change that

Reply:
https://flake8.pycqa.org/en/latest/ and https://github.com/psf/black tools come to mind.

Reply:
@proppy Will take that into account.